cmd/swapd/contract.go

```diff
@@ -8,19 +8,20 @@ import (
 	"fmt"
 	"path"
 
+	gsnforwarder "github.com/AthanorLabs/go-relayer/examples/gsn_forwarder"
```

Comment:
Is this our code? It seems weird to be importing from a package that includes a folder named `examples` and then a package name with an underscore.  Underscores can be used, but it's considered non-idiomatic: https://go.dev/doc/effective_go#package-names

Reply:
good point, updated!